tests/python_tests/test_llm_pipeline.py

```diff
@@ -121,20 +122,24 @@ def test_empty_encoded_inputs_throw():
     'What was my first question?'
 ]
 
-@pytest.mark.parametrize("generation_config_kwargs", generation_configs)
+@pytest.mark.parametrize("intpus", chat_intpus)
 @pytest.mark.parametrize("model_descr", get_chat_models_list())
 @pytest.mark.precommit
 @pytest.mark.nightly
-def test_chat_scenario(model_descr, generation_config_kwargs: Dict):
+def test_chat_scenario(model_descr, intpus):
     chat_history_hf = []
     chat_history_ov = []
 
     model_id, path, tokenizer, opt_model, ov_pipe = read_model((model_descr[0], model_descr[1]))
 
+    generation_config_kwargs, system_massage = intpus
+
     ov_generation_config = GenerationConfig(**generation_config_kwargs)
     hf_generation_config = convert_to_hf(opt_model.generation_config, ov_generation_config)
 
-    ov_pipe.start_chat()
+    ov_pipe.start_chat(system_massage)
+    chat_history_hf.append({"role", "system", "content", system_massage})
+    chat_history_ov.append({"role", "system", "content", system_massage})
```

Comment:
```suggestion
    chat_history_hf.append({"role": "system", "content": system_massage})
    chat_history_ov.append({"role:, "system", "content": system_massage})
```
You also got conflicts